cmd/vulcan-nuclei/main.go

```diff
@@ -134,16 +134,10 @@ func processNucleiFindings(target string, nucleiFindings []ResultEvent) []*repor
 	vulnerabilities := []*report.Vulnerability{}
 	rv := make(map[string]*report.Vulnerability)
 	for _, v := range nucleiFindings {
-		// Create resources table row.
-		// Avoid store redundant information in the resources table.
-		matched := v.Matched
-		if matched == target {
-			matched = ""
-		}
```

Comment:
What's the reason to remove this code?

Reply:
The reason is that after the change below the `matched` variable was not being used. I will explain why I decided in favor of this change in the other comment you wrote.